src/promptflow/promptflow/_sdk/_service/apis/connection.py

```diff
@@ -48,6 +53,15 @@
 )
 
 
+def get_connection_operation(working_directory=None):
+    if working_directory:
+        with _change_working_dir(working_directory):
```

Comment:
This might be error prone for a long-live service. Could we avoid changing working dir?

Reply:
remove changing working dir